workers/cpp_physics/physics.cc

```diff
@@ -0,0 +1,104 @@
+#include <algorithm>
+#include <chrono>
+#include <iostream>
+#include <string>
+#include <thread>
+#include <cmath>
+
+#include <improbable/worker.h>
+#include <improbable/view.h>
+
+#include <improbable/standard_library.h>
+#include <client_data.h>
+
+using ComponentRegistry = worker::Components<improbable::Position, improbable::Metadata>;
+
+const int kErrorExitStatus = 1;
+const std::uint32_t kGetOpListTimeoutInMilliseconds = 100;
+
+int main(int argc, char** argv) {
+    auto print_usage = [&]() {
+        std::cout << "Usage: Physics <hostname> <port> <worker_id>" << std::endl;
+        std::cout << std::endl;
+        std::cout << "Connects to SpatialOS" << std::endl;
+        std::cout << "    <hostname>      - hostname of the receptionist to connect to." << std::endl;
+        std::cout << "    <port>          - port to use if connecting through the receptionist." << std::endl;
+        std::cout << "    <worker_id>     - (optional) name of the worker assigned by SpatialOS." << std::endl;
```

Comment:
It's not optional since you require 4 parameters below

Reply:
lol, yes, I forgot to update this :P